lib/pubsub/topic.js

```diff
@@ -276,7 +313,10 @@ Topic.prototype.subscribe = function(name, options, callback) {
 
   var path = Subscription.formatName_(this.projectId, name);
   this.makeReq_('PUT', path, null, body, function(err) {
-    if (err) {
+    if (options.reuseExisting && err && err.code === 409) {
```

Comment:
It's not in the tests. Honestly, the fallback path is to call topic.subscription(). We can add a test that'll throw a 409 but it's hard to tell if .subscription was called from the 409 branch or the 200 branch.